ppsspp_config.h

```diff
@@ -64,6 +64,12 @@
     #define PPSSPP_ARCH_32BIT 1
 #endif
 
+#if defined(__riscv) && defined(__riscv_xlen) && __riscv_xlen == 64
+    //https://github.com/riscv/riscv-c-api-doc/blob/master/riscv-c-api.md
```

Comment:
nit: Oops, spaces in a land of tabs.

-[Unknown]

Reply:
Apologies!
IT might be worthwhile looking at including an .editorconfig file in the repo? https://editorconfig.org/

Reply:
There is one: https://github.com/hrydgard/ppsspp/blob/86c5a17d1e3e27aee8c80e8cfebbb8173dba085b/.editorconfig#L6

-[Unknown]